src/main/java/com/epam/idea/rest/controller/TagController.java

```diff
@@ -1,7 +1,38 @@
 package com.epam.idea.rest.controller;
 
+import java.util.List;
+
+import com.epam.idea.core.model.Tag;
+import com.epam.idea.core.service.TagService;
+import com.epam.idea.rest.resource.TagResource;
+import com.epam.idea.rest.resource.asm.TagResourceAsm;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 
 @Controller
+@RequestMapping("/api/v1/tags")
 public class TagController {
+
+    @Autowired
+    private TagService tagService;
+
+    @RequestMapping(value = "/{tagId}", method = RequestMethod.GET)
+    public HttpEntity<TagResource> showTag(@PathVariable final long tagId) {
+        Tag foundTag = tagService.findOne(tagId);
```

Comment:
[A field of a class should always be referenced using this.](https://github.com/EPAMMogilev/idea/wiki/Java-Code-Style-Guideline#field-and-method-references)